DeviceCode/Targets/Native/STM32F4/DeviceCode/STM32F4_SPI/STM32F4_SPI_functions.cpp

```diff
@@ -59,7 +60,9 @@ void CPU_SPI_Uninitialize()
 BOOL CPU_SPI_nWrite16_nRead16( const SPI_CONFIGURATION& Configuration, UINT16* Write16, INT32 WriteCount, UINT16* Read16, INT32 ReadCount, INT32 ReadStartOffset )
 {
     NATIVE_PROFILE_HAL_PROCESSOR_SPI();
-    if(!CPU_SPI_Xaction_Start( Configuration )) return FALSE;
+    if(!CPU_SPI_Xaction_Start( Configuration ))
+        return FALSE;

```

Comment:
I disagree, the redundant braces are distracting for trivial statements and don't really contribute to readability.